### DISCUSSION TITLE
Is it possible to disable token streaming of responses in the UI?

### DISCUSSION THREAD

**Discussion by agentk** [open-webui/open-webui#4318] [General]
**Is your feature request related to a problem? Please describe.**
I'm envious of the singular response to chat prompts in on the ollama cli. I'd like the option to do the same in the UI.

**Describe the solution you'd like**
Ideally it would be a UI option but would also setting for something that can be added config.json.

**Describe alternatives you've considered**
Well, I've considered ditching the UI.

**Additional context**
Don't know if there is much to add.

**Comment by DaWe35** [comment]
Yes, you can disable streaming for each model on the Admin Panel:
![image](https://github.com/user-attachments/assets/9b755420-2073-4af3-9f65-88070ac51b32)
